docs/area-owners.md

```diff
@@ -2,22 +2,22 @@ The below table lists all the `area-`labels used in the `aspnetcore` repository
 
 | Area label | Owner | Description|
 |--- | ---| --- |
-| area-azure | anurse | |
+| area-azure | BrennanConroy | |
 | area-blazor | mkArtakMSFT | Blazor server and Blazor WASM related |
-| area-commandlinetools | anurse, mkArtakMSFT | dev certs, dotnet watch,  |
-| area-dataprotection | anurse | |
+| area-commandlinetools |  mkArtakMSFT | dev certs, dotnet watch,  |
+| area-dataprotection | BrennanConroy | |
 | area-grpc | shirhatti | |
-| area-healthchecks | rynowak | |
-| area-hosting | anurse | |
-| area-httpclientfactory | anurse | |
+| area-healthchecks | mkArtakMSFT | |
+| area-hosting | BrennanConroy | |
+| area-httpclientfactory | mkArtakMSFT | |
 | area-identity | blowdart | |
 | area-infrastructure | dougbu | |
-| area-installers | anurse | |
-| area-middleware | anurse | |
+| area-installers | dougbu | |
+| area-middleware | BrennanConroy | |
 | area-mvc | mkArtakMSFT | |
-| area-perf | anurse | |
-| area-platform | anurse | |
+| area-perf | sebastienros | |
+| area-platform | BrennanConroy | |
```

Comment:
This one is being monitored by @JunTaoLuo

Reply:
Yea this should be me.

Reply:
```suggestion
| area-platform | JunTaoLuo | |
```